components/tag/TagsInput.js

```diff
@@ -11,10 +18,22 @@ export default function TagsInput({ tags, onTagAdd, onTagRemove }) {
     backspace: 8,
   };
 
+  const maxTagChar = 32;
```

Comment:
I think this can be renamed to `maxLength` because it is already in the tag component

Reply:
ok